packages/it-multipart/test.js

```diff
@@ -26,7 +34,9 @@ test.before.cb((t) => {
 
   server = http.createServer((req, res) => {
     if (req.method === 'POST' &&
+      // @ts-ignore - header may not be present
```

Comment:
If it's not present this will error so it should null-guard

Reply:
I did not bother because it's a test and all request will have it.